pom.xml

```diff
@@ -6,157 +6,222 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.dogtagpki</groupId>
     <artifactId>pki</artifactId>
-    <version>11.0.0-SNAPSHOT</version>
+    <version>11.3.0-SNAPSHOT</version>
```

Comment:
This version is repeated in all pom files. I think it is better to define a propertiy section [1] here and use it everywhere to simplify the management of the versions.
Other approaches are possible [2].

[1] https://maven.apache.org/pom.html#Properties
[2] https://stackoverflow.com/questions/17312681/maven-multi-module-project-version-management

Reply:
Agreed!

Reply:
So apparently there are different ways to manage the versions. One of the recommended methods is actually to use `mvn versions:set` to hard-code the versions into all `pom.xml` files just like in this PR originally, but I've updated the PR to use properties described here: https://maven.apache.org/maven-ci-friendly.html. Could you take a look again? Thanks!